apps/emqx/src/bpapi/emqx_bpapi.erl

```diff
@@ -0,0 +1,39 @@
+%%--------------------------------------------------------------------
+%% Copyright (c) 2021 EMQ Technologies Co., Ltd. All Rights Reserved.
+%%
+%% Licensed under the Apache License, Version 2.0 (the "License");
+%% you may not use this file except in compliance with the License.
+%% You may obtain a copy of the License at
+%%
+%%     http://www.apache.org/licenses/LICENSE-2.0
+%%
+%% Unless required by applicable law or agreed to in writing, software
+%% distributed under the License is distributed on an "AS IS" BASIS,
+%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%% See the License for the specific language governing permissions and
+%% limitations under the License.
+%%--------------------------------------------------------------------
+-module(emqx_bpapi).
+
+-export_type([var_name/0, call/0, rpc/0, bpapi_meta/0]).
+
+-type api() :: atom().
+-type api_version() :: non_neg_integer().
```

Comment:
Just curiosity: since those are used in the [internal state](https://github.com/emqx/emqx/pull/6629/files#diff-048bea61123ebd1af9e3cbfee089790dc190d5d701c2effc86995bcf86f3a675R29) of the parse transformation, would it make sense to export those types and use them there?

Reply:
Makes sense. I will fix this in the follow-up